lnrpc/wtclientrpc/wtclient.proto

```diff
@@ -113,6 +131,12 @@ message Tower {
 message ListTowersRequest {
     // Whether we should include sessions with the watchtower in the response.
     bool include_sessions = 1;
+
+    /*
+    If true, will submit the request to the anchor channel client, otherwise the
+    request will be submitted to the legacy channel client.
+    */
+    bool anchor_client = 2;
```

Comment:
Looking a bit further ahead in the future, we could alliteratively have this be an enum for what type of client it is based on the channel type. Ideally we can order things in a way that doesn't put too much burden on the caller as well.

Reply:
added enum. also unified the other rpcs